CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ CHANGELOG
 
 3.3.1
 -----
+
+**ENHANCEMENTS**
+- Add `UpdateReplacePolicy` attribute to EFS, EBS and FSx for Lustre shared storage to support storage retention in case
+  of resource replacement. The value of the new policy is set through the configuration parameter `DeletionPolicy`.
```

Comment:
What about describing te change more from a user perspective: When setting `DeletionPolicy` to `Retain`, the file system, configured in the `SharedStorage` section, is now preserved also during a cluster update operation.

Reply:
changed